Runtime-Implementation-Details/Relational-Storage.md

```diff
@@ -2,33 +2,112 @@
 layout: page
 title: Relational Storage
 ---
+Relational storage backend code in Orleans is built on generic ADO.NET functionality and is consequently database vendor agnostic. The Orleans data storage layout is explained in [Runtime Tables](Runtime-Tables). Setting up the connection strings is explained in [Orleans Configuration Guide](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/) and [SQL Tables](http://dotnet.github.io/orleans/Advanced-Concepts/Configuring-SQL-Tables).
 
-Relational storage backend code in Orleans is built on generic ADO.NET functionality and is consequently database vendor agnostic. The Orleans data storage layout has been explained already in [Runtime Tables](Runtime-Tables). Setting up the connection strings are done as explained in [Orleans Configuration Guide ](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/) and [SQL Tables](http://dotnet.github.io/orleans/Advanced-Concepts/Configuring-SQL-Tables).
+To make Orleans code function with a given relational database backend, the following is required:
 
-To make Orleans code function with a given relational database backend, two things are needed:
+1. Appropriate ADO.NET library must be loaded to the process (multiple can exist in GAC or otherwise). This should be defined as usual, e.g. via [DbProviderFactories](https://msdn.microsoft.com/en-us/library/dd0w4a2z(v=vs.110).aspx) element in application configuration.
+2. Configure the ADO.NET invariant via ``AdoInvariant`` attribute in the element defining the connection string, by default it is `System.Data.SqlClient`
```

Comment:
Perhaps more explicit would to write "if not given, by default it is...". Maybe, I'm not sure, but would expound on the point, which might become easier to those who do not understand English well.